cmd/application-create.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/redhat-developer/ocdev/pkg/ocdev"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var createCmd = &cobra.Command{
```

Comment:
could we have this in app.go together?

Reply:
I tried, but cobra needs to have an init() function for every new command, so I didn't figure out a way on how to do that :(

Reply:
look at this : https://github.com/kadel/ocdev/blob/componentCreate/cmd/component.go

I've  put command and subcommand into one file.

Reply:
Hmm, works now, was failing for me before :man_shrugging: 
works now!